contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.3.0--2.4.0.sql

```diff
@@ -0,0 +1,41 @@
+-- complain if script is sourced in psql, rather than via ALTER EXTENSION
+\echo Use "ALTER EXTENSION ""babelfishpg_tsql"" UPDATE TO '2.4.0'" to load this file. \quit
+
+-- add 'sys' to search path for the convenience
+SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false);
+
+-- Drops a view if it does not have any dependent objects.
+-- Is a temporary procedure for use by the upgrade script. Will be dropped at the end of the upgrade.
+-- Please have this be one of the first statements executed in this upgrade script. 
+CREATE OR REPLACE PROCEDURE babelfish_drop_deprecated_view(schema_name varchar, view_name varchar) AS
```

Comment:
Let's use `babelfish_drop_deprecated_object()` function so that we can use this to drop view/function/procdures.